index.js

```diff
@@ -29,15 +29,23 @@ if (process.env.AWS_ACCESS_KEY_ID) {
 }
 
 if (process.env.SLACK_KEY) {
-  ['SLACK_SECRET', 'SLACK_VERIFICATION_TOKEN', 'SLACK_APP_ID'].forEach(key => {
-    if (!process.env[key]) {
-      console.error(
-        `The ${key} env variable must be set when using the Slack integration`
-      );
-      // $FlowFixMe
-      process.exit(1);
-    }
-  });
+  if (!process.env.SLACK_SECRET) {
+    console.error(
+      `The SLACK_SECRET env variable must be set when using Slack Sign In`
+    );
+    // $FlowFixMe
+    process.exit(1);
+  }
+}
+
+if (process.env.SLACK_APP_ID) {
+  if (!process.env.SLACK_VERIFICATION_TOKEN) {
+    console.error(
+      `The SLACK_VERIFICATION_TOKEN env variable must be set when using the Slack slash command`
+    );
+    // $FlowFixMe
+    process.exit(1);
+  }
 }
```

Comment:
To be honest after reassessing the code `SLACK_APP_ID` is not needed for the slash command to work either, I think we can just remove this entire block

Reply:
Cool! I'll make that update